doc/attributes.md

```diff
@@ -67,3 +67,50 @@ message:
 
 When an attribute is used in Istio, its name is given along with a value. The value must be of the type declared by the corresponding descriptor. This
 type checking makes it possible for the Istio system to statically detect or prevent many configuration errors.
+
+# Standard Istio Attribute Vocabulary
+
+In a standard Istio deployment (using the Istio proxy), the system will produce the following set of attributes. This is considered the standard, or canonical, attribute set for Istio.
+
+| Name| Type| Description| Kubernetes Example| HTTP Header Name| Notes|
+|-----|-----|------------|-------------------|-----------------|------|
+| source.ip| ip_address| The IP address of the client that sent the request.| 10.0.0.117| | |
+| source.port| int64| The port on the client that sent the request.| 9284| | |
+| source.name| string| The fully qualified name of the application that sent the request.| redis-master.namespace.deps.cluster.local| x-envoy-downstream-service-cluster| Using a name this is not a resolvable DNS name. MIght need to tweak to avoid collision|
+| source.uid| string| A unique identifier for the particular instance of the application that sent the request.| redis-master-2353460263-1ecey.namespace.pods.cluster.local| | |
+| source.namespace| string| The namespace of the sender of the request.| namespace.cluster.local| | |
+| source.labels| map| A map of key-value pairs attached to the source.| | | |
```

Comment:
Let's add examples where we can and they are missing.

Happy to take a pass in a separate PR.